source/stylesheets/modules/_anchored-heading.scss

```diff
@@ -1,14 +1,37 @@
+$anchored-heading-icon-width: 23px;
+$anchored-heading-icon-height: 14px;
+$anchored-heading-icon-spacing: 3px;
+
 .anchored-heading {
   position: relative;
 }
 
 .anchored-heading__icon {
   position: absolute;
+  width: $anchored-heading-icon-width;
+  left: -($anchored-heading-icon-width + $anchored-heading-icon-spacing);
+  padding-right: $anchored-heading-icon-spacing;
+  margin-top: -1px;
+
   display: none;
+
   text-decoration: none;
-  left: -1em;
+  text-indent: -9999em;
+
+  background-image: image-url('anchored-heading-icon.png');
```

Comment:
IE support is fairly shonky – in 9-11 they don't scale particularly well, and in 6-8 they're not supported at all. We could default to SVG and then use a PNG as a fallback, but the SVG is marginally larger in terms of file-size, even when compared to the retina PNG.